voice/bxml/verbs/record.md

```diff
@@ -9,7 +9,7 @@ The Record verb allows a segment of audio to be recorded during a call. At the e
 | recordCompleteMethod         | (optional) The HTTP method to use for the request to `recordCompleteUrl`. GET or POST. Default value is POST.                                                                                                          |
 | recordingAvailableUrl        | (optional) URL to send the [Recording Available](../callbacks/recordingAvailable.md) event to once it has been processed. Does not accept BXML.                                                                        |
 | recordingAvailableMethod     | (optional) The HTTP method to use for the request to `recordingAvailableUrl`. GET or POST. Default value is POST.                                                                                                      |
-| transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Default is `false`.                                                                                                                       |
+| transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Transcription can succeed only for recordings of length greater than 500 milliseconds and less than 4 hours. Default is `false`.                                                                                                                       |
```

Comment:
Don't need to change the words, but can you change the number of spaces at the end so the table lines up? Doesn't matter to customers, but it's easier to read for us.
If you do change the words though, make sure to change them in each of these files.
```suggestion
| transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Transcription can succeed only for recordings that are longer than 500 milliseconds and less than 4 hours. Default is `false`.          |
```

Reply:
Also `silenceTimeout` has 2 extra spaces on its line (though you didn't add them, might as well remove them).